components/gitpod-db/src/typeorm/migration/1592203031938-Baseline.ts

```diff
@@ -60,7 +60,7 @@ export class Baseline1592203031938 implements MigrationInterface {
                     { url: 'https://github.com/gitpod-io/django-locallibrary-tutorial', description: '**Python** - Tutorial "Local Library" website written in Django', priority: 10 },
                     { url: 'https://github.com/gitpod-io/gs-spring-boot.git', description: '**Java** - Building an Application with Spring Boot', priority: 9 },
                     { url: 'https://github.com/gitpod-io/symfony-demo.git', description: '**PHP** - Symfony Demo Application', priority: 5 },
-                    { url: 'https://github.com/theia-ide/theia.git', description: '**Theia** - Deep dive into Gitpods open-source IDE. (TypeScript)', priority: 4 }
+                    { url: 'https://github.com/theia-ide/theia.git', description: "**Theia** - Deep dive into Gitpod's open-source IDE (written in TypeScript).", priority: 4 }
```

Comment:
It would be better to align the naming structure as the rest of these entries. WDYT @Rexogamer?

```suggestion
                    { url: 'https://github.com/theia-ide/theia.git', description: "**TypeScript** - Deep dive into Gitpod's open-source IDE", priority: 4 }
```

Reply:
How about:
```suggestion
                    { url: 'https://github.com/theia-ide/theia.git', description: "**Typescript** - Deep dive into Gitpod's open-source IDE, Theia..", priority: 4 }
```

Reply:
Thanks @Rexogamer! FWIW, I find the repeated term (_Theia_) on the list description slightly redundant and the suggestion in https://github.com/gitpod-io/gitpod/pull/2145#discussion_r523799673 will introduce a puncuation inconsistency in this list. Having content guidelines could certainly help in the future, however, this page will probably get a lot better soon.

Let's merge this now and iterate on making it better! 🚀 

/cc @jankeromnes because they have previously worked on updating this list.